src/components/DryRunnerCardLog/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import Column from '../Column'
+import Image from '../Image'
+import Row from '../Row'
+import arrayPad from '../../helpers/arrayPad'
+import './index.css'
+
+const DryRunnerCardLog = props => {
```

Comment:
Let’s wrap it with `React.memo(…)`. I did that on almost all pure components to try to improve performance.